src/peergos/server/Main.java

```diff
@@ -433,8 +433,10 @@ public static UserService startPeergos(Args a) {
             JdbcSpaceRequests spaceRequests = JdbcSpaceRequests.build(spaceDb, sqlCommands);
             UserQuotas userQuotas = new UserQuotas(quotaFilePath, defaultQuota, maxUsers);
             CoreNode signupFilter = new SignUpFilter(core, userQuotas, nodeId);
+            RamUsageStore usageStore = RamUsageStore.build(statePath, localPointers, localDht);
+            SpaceCheckingKeyFilter.update(usageStore, localPointers, localDht);
```

Comment:
This update method 'feels' wrong.  How about turning into a builder method?

Reply:
I get what you're saying, but it is updating the calculated usage in case any of the pointers have changed.